local_node.sh

```diff
@@ -212,5 +212,6 @@ evmosd start \
 	--log_level $LOGLEVEL \
 	--minimum-gas-prices=0.0001aevmos \
 	--json-rpc.api eth,txpool,personal,net,debug,web3 \
+	--json-rpc.allow-insecure-unlock true \
```

Comment:
feel free to add this on a per-need basis, it shouldn't be the default here

```suggestion
```

Reply:
Previously, It used init-node.sh under e2e, but because e2e has TestUpgrade, it will use the old evmos binary file. If  add “--json-rpc.allow-insecure-unlock true” directly to e2e, an error will occur.